spec/features/projects/projects_index_spec.rb

```diff
@@ -202,6 +202,21 @@ def expect_projects_in_order(*projects)
         end
       end
 
+      specify 'long description is truncated' do
+        Setting.enabled_projects_columns += ['description']
+
+        development_project.update(
+          description: 'I am a nice project with a very long long long long long long long long long description'
+        )
+
+        login_as(admin)
+        projects_page.visit!
+        tr_project_development = page.first('tr', text: 'Development project')
+        expect(tr_project_development).to have_css('button', text: 'Expand')
+        tr_project_development.find('button', text: 'Expand').click
+        expect(page).to have_css('.Overlay-body', text: development_project.description)
+      end
```

Comment:
I'd add a case to this test for the other types of fields as well: status explanation and long text custom fields. I'd also add a negative case test in which you check that the "Expand" button is not displayed on shorter text.